src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -709,6 +718,17 @@ export class Workbench implements IPartService {
 		// Create Workbench DIV Off-DOM
 		this.workbenchContainer = $('.monaco-workbench-container');
 		this.workbench = $().div({ 'class': 'monaco-workbench ' + (isWindows ? 'windows' : isLinux ? 'linux' : 'mac'), id: Identifiers.WORKBENCH_CONTAINER }).appendTo(this.workbenchContainer);
+
+		// Only needed while the option `windowConfig.macOSUseInlineToolbar` exists
+		if (isMacintosh) {
```

Comment:
Please move all of this code into `renderWorkbench` where we already adjust some CSS classes based on things like sidebar visible or not. It is also much easier to get hold of the IConfigurationService by adding the following to the constructor of the workbench:

`@IConfigurationService private configurationService: IConfigurationService`

All the other code is not needed then. 

To find out if the setting is set, I suggest this code:

`this.configurationService.lookup('window.macOSUseInlineToolbar').value`